app/src/models/User.py

```diff
@@ -1,9 +1,10 @@
 from app.src.utilities.logger import logger
 from typing import List, Any
+from flask_login import UserMixin
 import json
 
 
-class User:
+class User(UserMixin):
```

Comment:
Whats Mixin? and whats the type of that variable

Reply:
UserMixin is a class provided by flask-login module, and my User class inherits from it and I don't have to implement methods that are necessary for that module to work.

Reply:
arigato thomasa sensei